rust/config/testnet4_config.json

```diff
@@ -0,0 +1,616 @@
+{
+  "chains": {
+    "alfajores": {
+      "chainId": 44787,
+      "domainId": 44787,
+      "name": "alfajores",
+      "protocol": "ethereum",
+      "displayName": "Alfajores",
+      "nativeToken": {
+        "decimals": 18,
+        "name": "CELO",
+        "symbol": "CELO"
+      },
+      "rpcUrls": [
+        {
+          "http": "https://alfajores-forno.celo-testnet.org"
+        }
+      ],
+      "blockExplorers": [
+        {
+          "name": "CeloScan",
+          "url": "https://alfajores.celoscan.io",
+          "apiUrl": "https://api-alfajores.celoscan.io/api",
+          "family": "etherscan"
+        },
+        {
+          "name": "Blockscout",
+          "url": "https://explorer.celo.org/alfajores",
+          "apiUrl": "https://explorer.celo.org/alfajores/api",
+          "family": "blockscout"
+        }
+      ],
+      "blocks": {
+        "confirmations": 1,
+        "reorgPeriod": 0,
+        "estimateBlockTime": 5
+      },
+      "isTestnet": true,
+      "merkleRootMultisigIsmFactory": "0xa9C7e306C0941896CA1fd528aA59089571D8D67E",
```

Comment:
Not for the scope of this PR but having all these unused addresses in the Rust config feels like something we should change at some point

I never deeply reviewed the config change PR so that's on me for not flagging earlier